mlir/utils/jenkins/Jenkinsfile.downstream

```diff
@@ -43,6 +43,13 @@ pipeline {
 
                     # build LLVM / MLR and run tests
                     cmake --build . --target check-mlir
+
+	     	    # premerge checks
+                    cd ..
+                    if [ ! -f ./compile_commands.json ]; then
+                       ln -s build/compile_commands.json compile_commands.json
+                    fi
```

Comment:
Why do you need this symlink?

Reply:
Clang-tidy needs this file and it has to be at the llvm-project-mlir directory.

Reply:
Is this because of `clang-tidy` being not invoked from the `build` directory? Can it be invoked in `build` directory?

Reply:
No, it gave file-not-found kind of errors. And Gibrater does the symlink as well.

Reply:
Sounds good.